app/src/main/res/values/strings.xml

```diff
@@ -746,4 +756,4 @@
     <string name="ir_thermometer_desc">\u2022 The above pin configuration has to be same except for the pin
     GND. You can use any of the PSLab device GND pins.\n</string>
 
-</resources>
+</resources>
```

Comment:
Here also. @Avjeet , @abhinavraj23 , @harsh-2711 , @Vikum94 Guys please don't remove this system generated new line character at each file explicitly.

Reply:
Noted